scripts/Muon/GUI/Common/corrections_tab_widget/background_corrections_view.py

```diff
@@ -0,0 +1,268 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2021 ISIS Rutherford Appleton Laboratory UKRI,
+#   NScD Oak Ridge National Laboratory, European Spallation Source,
+#   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+# SPDX - License - Identifier: GPL - 3.0 +
+from mantidqt.utils.qt import load_ui
+
+from qtpy.QtCore import Qt
+from qtpy.QtGui import QDoubleValidator
+from qtpy.QtWidgets import QLineEdit, QStyledItemDelegate, QTableWidgetItem, QWidget
+
+ui_form, widget = load_ui(__file__, "background_corrections_view.ui")
+
+RUN_COLUMN_INDEX = 0
+GROUP_COLUMN_INDEX = 1
+START_X_COLUMN_INDEX = 2
+END_X_COLUMN_INDEX = 3
+A0_COLUMN_INDEX = 4
+A0_ERROR_COLUMN_INDEX = 5
+
+
+class DoubleItemDelegate(QStyledItemDelegate):
+    """
+    An item delegate that has a QDoubleValidator to validate the data provided.
+    """
```

Comment:
shouldnt this have an init?

Reply:
It doesn't require any modification to the default init

Reply:
can we have a note?